agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/diagnostics/DiagnosticsHelper.java

```diff
@@ -39,11 +39,13 @@ private DiagnosticsHelper() {}
 
   private static final boolean isWindows;
 
+  public static final String LINUX_DEFAULT = "/var/log/applicationinsights";
```

Comment:
should we move this constant to StatusFile where it's used?

Reply:
it's also used in LoggingConfigurator.java now.